src/pages/execution/tabLoader.tsx

```diff
@@ -255,6 +265,76 @@ export default function tabLoader(tabBase: TabBase): TabData {
   };
 }
 
+function AlertComponent() {
+  const executionAndTasks = useContext(TileFactoryContext).executionAndTasks;
+  const { severity, setSeverity, setHasAlerts } =
```

Comment:
I don't think we even need `severity` here at all. `AlertComponent` only ever passes the max severity upstream (via `setSeverity`), no need to read it back.